### ISSUE TITLE
Tasks for launch of Jekyll site

### ISSUE THREAD

**Issue by trentmwillis** [qunitjs/qunitjs.com#151]
The following items still need to be addressed before launching:

- [x] Responsive/cross-browser styles (ensure the styles look good across most major browsers and viewport sizes)
  - [x] Fix iframe on mobile
- [x] Accessibility (mainly for the nav, most others areas should be good)
  - [x] Run axe-core audit on all the pages
- [x] Setup search (through Algolia which ESLint has been using)
- [x] Favicon
- [x] Point qunitjs.com domain to qunitjs.github.io

**Comment by Krinkle** [comment]
... and we're live!

| New | Old
|--|--
|  [<img width="1438" alt="New quntjs dot com" src="https://user-images.githubusercontent.com/156867/86847399-2b30a800-c0a4-11ea-85f4-6e9e6f8fb673.png">](https://qunitjs.com) | <img width="1438" alt="Old qunitjs dot com" src="https://user-images.githubusercontent.com/156867/86847417-34217980-c0a4-11ea-8a78-43acec705f0f.png">

**Comment by trentmwillis** [comment]
@Krinkle thank you so much for carrying this work over the finish line! Really happy to see this be live 💯 😄
